tests/testcases/mod.rs

```diff
@@ -46,6 +46,10 @@ pub async fn key_value_works(controller: &dyn Controller) {
         .name("key-value".to_string())
         .appname(Some("key-value".to_string()))
         .template(None)
+        .up_args(vec![
```

Comment:
I am anxious about this duplication for the cases where we are able to keep the syntax the same.

I wonder if it would be practical to have something like a `shared_args` (bad name though) which is applied to both `up` _and_ `deploy`, and use `up_args` and `deploy_args` only for target-specific cases like `--variables`?

Reply:
the reason i like the duplication is that i think we have discussed removing many of the deploy args in the future. This way, if that happens, it is clear what was up and what was deploy.